android/java/org/chromium/chrome/browser/vpn/wireguard/WireguardServiceImpl.java

```diff
@@ -152,8 +153,8 @@ private void recordSessionTimes() {
     private void updateRecordSessionTimesTimer() {
         // This timer will be active for the duration of the connection.
         // It will become inactive when the connection is terminated/service destroyed.
-        mVpnStatisticsTimer = new Timer();
-        mVpnStatisticsTimer.schedule(new TimerTask() {
+        mVpnRecordStatisticsTimer = new Timer();
```

Comment:
We need to use different object for timer as using the same object was affecting the notification update for the VPN.

Reply:
My mistake, I meant to use `mRecordDaysUsedTimer`. I will remove `mRecordDaysUsedTimer` in a new commit in this branch.